tests/unit/test_parsers.py

```diff
@@ -87,11 +87,6 @@
         #     Path(dir_path, "data", "cogent", "cogent2_result.json"),
         # ),
         # Colt
-        (
-            ICalParserColt1,
-            Path(dir_path, "data", "colt", "colt1"),
```

Comment:
If we are removing this parser, why do we keep the test data file?

Reply:
Seems to be an old version. The test has been removed. The data file is still in the repo as the `colt1.ics` is an attachment to the `colt3.eml`.